Scenarios/FileActivation/FileActivationSample/FileActivation.vsimporter/FileActivation-WinStore10/Generated Files/App.g.h

```diff
@@ -1,27 +0,0 @@
-﻿#pragma once
-//------------------------------------------------------------------------------
-//     This code was generated by a tool.
-//
-//     Changes to this file may cause incorrect behavior and will be lost if
-//     the code is regenerated.
-//------------------------------------------------------------------------------
-
-#include "XamlTypeInfo.g.h"
```

Comment:
nope!

---
In reply to: [107052986](https://github.com/Microsoft/WinObjC-Samples/pull/19#discussion_r107052986) [](ancestors = 107052986)